docker-compose.yaml

```diff
@@ -0,0 +1,31 @@
+services:
+  db:
+    image: postgres:13.18-alpine3.20
+    restart: always
+    env_file:
+      - .env
+    ports:
+      - "5432:5432"
+    volumes:
+      - my_db:$PGDATA
+
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8001:8000"
+    volumes:
+#      - ./:/app

```

Comment:
The commented-out volume mapping for the application code (`- ./:/app`) should be reviewed. If you need live code updates during development, consider uncommenting it. However, for production, it's better to build the image with the code included to avoid potential issues.